### PR TITLE
Add help files for new variation related cmdlets

### PR DESCRIPTION
**Pull request by zhoyingms**

### CODE REVIEW

**Comment by zhoyingms** [comment]
@microsoft-github-policy-service agree company="Microsoft"

Thanks
Ying

**Comment by yogkumgit** [comment]
@JohanFreelancer9

File: Deploy-SPVariation.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+If the Recurse parameter is used, variations for subsites and pages are also created.
+
+## EXAMPLES
+
+### EXAMPLE
+```
+C:\PS> Deploy-SPVariation -Identity https://server/sites/pub/vhome/source/newsub
+```
+
+The example is to create new site variations on the source variation site specified by the Identity parameter for all target variations labels.
```
Comment (JohanFreelancer9) [inline Deploy-SPVariation.md:31]:
```suggestion
The example creates new site variations on the source variation site specified by the Identity parameter for all target variations labels.

```

File: Deploy-SPVariation.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (JohanFreelancer9) [inline Deploy-SPVariation.md:127]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).

```

File: Get-SPVariationJob.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+external help file: Microsoft.SharePoint.Publishing.dll-Help.xml
+Module Name: SharePointServer
+online version:
+schema: 2.0.0
+---
+
+# Get-SPVariationJob
+
+## SYNOPSIS
+Displays current status of Variations Propagate Page Job Definition and Variations Propagate Site Job Definition timer jobs that are located on the Timer Job Status page of the SharePoint Central Administration Web site.
```
Comment (JohanFreelancer9) [inline Get-SPVariationJob.md:11]:
```suggestion
Displays the current status of Variations Propagate Page Job Definition and Variations Propagate Site Job Definition timer jobs that are located on the Timer Job Status page of the SharePoint Central Administration website.

```

File: Get-SPVariationJob.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+
+## SYNTAX
+
+```
+Get-SPVariationJob -Identity <SPWebPipeBind> [-AssignmentCollection <SPAssignmentCollection>] [-WhatIf]
+ [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+The command displays current status of Variations Propagate Page Job Definition and Variations Propagate Site Job Definition timer jobs that are located on the Timer Job Status page of the SharePoint Central Administration Web site.
```
Comment (JohanFreelancer9) [inline Get-SPVariationJob.md:21]:
```suggestion
The command displays the current status of Variations Propagate Page Job Definition and Variations Propagate Site Job Definition timer jobs that are located on the Timer Job Status page of the SharePoint Central Administration website.

```

File: Get-SPVariationJob.md [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+
+## EXAMPLES
+
+### EXAMPLE
+```
+C:\PS> Get-SPVariationJob -Identity https://server/sites/pub/vhome/source/newsub
+```
+
+The example is to display the current status of Variations Propagate Page Job Definition and Variations Propagate Site Job Definition timer jobs that are located on the Timer Job Status page of the SharePoint Central Administration Web site.
```
Comment (JohanFreelancer9) [inline Get-SPVariationJob.md:30]:
```suggestion
This example displays the current status of Variations Propagate Page Job Definition and Variations Propagate Site Job Definition timer jobs that are located on the Timer Job Status page of the SharePoint Central Administration website.

```

File: Get-SPVariationJob.md [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (JohanFreelancer9) [inline Get-SPVariationJob.md:96]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).

```

File: Repair-SPVariation.md [hunk 7]
```diff
@@ -0,0 +1,135 @@
...
+If the recurse parameter is used, fixes are done recursively for all subsites.
+
+## EXAMPLES
+
+### EXAMPLE
+```
+C:\PS> Repair-SPVariation -Identity https://server/sites/pub/vhome/source/newsub -Recurse
+```
+
+The example is to correct variations system data for the source variation on site https://server/sites/pub/vhome/source/newsub and all its subsites.
```
Comment (JohanFreelancer9) [inline Repair-SPVariation.md:31]:
```suggestion
This example corrects variations system data for the source variation on site https://server/sites/pub/vhome/source/newsub and all its subsites.

```

File: Repair-SPVariation.md [hunk 8]
```diff
@@ -0,0 +1,135 @@
...
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (JohanFreelancer9) [inline Repair-SPVariation.md:127]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).

```

File: Test-SPVariation.md [hunk 9]
```diff
@@ -0,0 +1,134 @@
...
+The command analyzes the variations hierarchy and reports identified problems.
+
+## EXAMPLES
+
+### EXAMPLE
+```
+C:\PS> Test-SPVariation -Identity https://server/sites/pub/vhome/source/newsub
+```
+
+The example is to analyze the variations hierarchy and report identified problems.
```
Comment (JohanFreelancer9) [inline Test-SPVariation.md:30]:
```suggestion
This example analyzes the variations hierarchy and reports identified problems.

```

File: Test-SPVariation.md [hunk 10]
```diff
@@ -0,0 +1,134 @@
...
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (JohanFreelancer9) [inline Test-SPVariation.md:126]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).

```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @zhoyingms, kindly commit the suggested changes. Many thanks.

File: Get-SPVariationJob.md [hunk 11]
```diff
@@ -0,0 +1,104 @@
...
+```
+C:\PS> Get-SPVariationJob -Identity https://server/sites/pub/vhome/source/newsub
+```
+
+The example is to display the current status of Variations Propagate Page Job Definition and Variations Propagate Site Job Definition timer jobs that are located on the Timer Job Status page of the SharePoint Central Administration Web site.
+
+## PARAMETERS
+
+### -AssignmentCollection
+{{ Fill AssignmentCollection Description }}
```
[the remark this answers is not in the document]
Comment (zhoyingms) [reply Get-SPVariationJob.md]: Thanks for your suggestion. Have added.

File: Deploy-SPVariation.md (hunk 1 above)
Comment (zhoyingms) [reply Deploy-SPVariation.md]: Thanks for your suggestions. Have updated.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by dariomws — CHANGES_REQUESTED** [review]
@zhoyingms, should these files be stored in the root folder? I think they should be in:
officedocs-sharepoint-powershell/sharepoint/sharepoint-ps/sharepoint-server/

**Comment by zhoyingms** [comment]
@dariomws Thanks for your suggestions, I have updated the file locations.

**Comment by yogkumgit** [comment]
@dariomws @get-itips

**Review by dariomws — APPROVED** [review]

**Comment by yogkumgit** [comment]
@get-itips

**Review by get-itips — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**